components/cloud/azure/src/azblob.rs

```diff
@@ -0,0 +1,740 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+use async_trait::async_trait;
+use azure_core::auth::{TokenCredential, TokenResponse};
+use azure_core::prelude::*;
+use azure_identity::token_credentials::{ClientSecretCredential, TokenCredentialOptions};
+use azure_storage::blob::prelude::*;
+use azure_storage::core::{prelude::*, ConnectionStringBuilder};
+use chrono::{Duration as chDuration, Utc};
```

Comment:
The proper casing should be

```suggestion
use chrono::{Duration as ChDuration, Utc};
```